app/src/main/java/com/greenfox/gitinder/Constants.java

```diff
@@ -16,7 +18,17 @@
     public static final String GO_TO_MATCHES = "GoToMatches";
     public static final String PROFILE = "profile";
     public static final String USERNAME = "username";
+    public static final String GET_PROFILES_ENDPOINT = "Get Profiles";
+    public static final String SWIPING_ENDPOINT = "Swiping";
+    public static final String GET_MATCHES_ENDPOINT = "Get Matches";
+    public static final String GET_MESSAGES_ENDPOINT = "GET Messages";
+    public static final String SEND_MESSAGE_ENDPOINT = "Send Message";
+    public static final String GET_SETTINGS__ENDPOINT = "Get Settings";
+    public static final String SAVE_SETTINGS_ENDPOINT = "Save Settings";
+    public static final String LOGIN_ENDPOINT = "Login";
+    public static final String LOGOUT_ENDPOINT = "Logout";
+    public static final String SEEN_ENDPOINT = "Seen";
```

Comment:
@Splichus remove the _ENDPOINT endings, to make the readability a little bit simpler